api/v1/utils/roles.js

```diff
@@ -0,0 +1,10 @@
+var _ = require('lodash');
+
+var ALL_ROLES = ['ADMIN', 'STAFF', 'SPONSOR', 'MENTOR', 'VOLUNTEER', 'HACKER'];
```

Comment:
This should be brought up at a systems call of some sort. We had a call like this last year but it was done super late. Throw something on the calendar when we move on to that.